pkg/trace/sampler/coresampler.go

```diff
@@ -179,6 +187,9 @@ func (s *Sampler) updateRates(previousBucket, newBucket int64) {
 			delete(s.seen, sig)
 			continue
 		}
+		if rate < s.minRate {
```

Comment:
maybe renaming minRate to `lowestRate` would make it a bit easier to follow?
It's not a condition, it's an observation.